.git-blame-ignore-revs

```diff
@@ -0,0 +1,2 @@
+#ded44948e2d5a398abcb4e342b0513cb690961bb # Happy new year 2022 (#10573)
```

Comment:
Why is this commented out?

Reply:
I have added it first as something that is irrelevant to the actual history of the file. Then I noticed that it only affects a single line in the blame/annotation, so we can leave it in. And then I realized it might help people to understand the syntax of the file by example (as opposed to looking at the documentation which says "in the same format as an fsck.skipList").

Reply:
Either write a proper doc line or remove. You already have an example by the line below.

Reply:
Okay.